app/controllers/ReportingController.scala

```diff
@@ -110,6 +117,70 @@ trait ReportingController extends BaseController {
     }
   }
 
+  def streamPreviousYearCandidatesDetailsReport: Action[AnyContent] = Action.async { implicit request =>
+    enrichPreviousYearCandidateDetails {
+      (contactDetails, questionnaireDetails, mediaDetails, eventsDetails, siftAnswers) =>
+      {
+        val header = Enumerator(
+          (prevYearCandidatesDetailsRepository.applicationDetailsHeader ::
+            prevYearCandidatesDetailsRepository.contactDetailsHeader ::
+            prevYearCandidatesDetailsRepository.questionnaireDetailsHeader ::
+            prevYearCandidatesDetailsRepository.mediaHeader ::
+            prevYearCandidatesDetailsRepository.eventsDetailsHeader ::
+            prevYearCandidatesDetailsRepository.siftAnswersHeader ::
+            Nil).mkString(",") + "\n"
+        )
+        var counter = 0
+        val candidatesStream = prevYearCandidatesDetailsRepository.applicationDetailsStream().map { app =>
+          val ret = createCandidateInfoBackUpRecord(app, contactDetails, questionnaireDetails, mediaDetails, eventsDetails, siftAnswers) + "\n"
+          // Logger.debug("Output line " + counter + ": " + ret)
+          counter += 1
+          ret
+        }
+        Ok.chunked(Source.fromPublisher(Streams.enumeratorToPublisher(header.andThen(candidatesStream))))
+      }
+    }
+  }
+
+  // scalastyle:off line.size.limit
```

Comment:
This required? Lines look ok to me. Also, I think we can always just re-structure the lines rather than silencing the warning

Reply:
+